docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -1,36 +1,61 @@
 # SGX1 with Flexible Launch Control (FLC) Prerequisites on Windows
 
-## [Intel Platform Software for Windows (PSW) v2.12](https://registrationcenter-download.intel.com/akdlm/irc_nas/17361/Intel%20SGX%20PSW%20for%20Windows%20v2.12.100.4.exe)
+## Intel Platform Software for Windows (PSW)
+
+Intel PSW only needs to be manually installed if you are running a version of Windows client lower than 1709. It should be installed automatically with Windows Update on newer versions of Windows client and Windows Server 2019. You can check your version of Windows by running `winver` on the command line.
 
-The PSW only needs to be manually installed if you are running a version of Windows client lower than 1709. It should be installed automatically with Windows Update on newer versions of Windows client and Windows Server 2019.
-You can check your version of Windows by running `winver` on the command line.
 Ensure that you have the latest drivers on Windows 10 and Windows Server 2019 by checking for updates and installing all updates.
 
-To install the PSW on Windows client < 1709, unpack the self-extracting
-ZIP executable, and run the installer under `PSW_EXE_RS2_and_before`:
+### Manual installation for client versions lower than 1709
+To manually install Intel PSW on Windows for client versions lower than 1709:
+
+1. Download the latest Intel SGX PSW via Intel Developer Zone, or use the direct download link specified by `$IntelPSWURL` in [install-windows-prereqs.ps1](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1#L17). In this specific example we will use Intel SGX PSW for Windows v2.12.100.4.
```

Comment:
So when it says "the latest", we are guaranteeing that it will never be the case that Intil might release a newer version before OE can be updated to support it?

Reply:
Good question. Since we do get the opportunity to test and raise compatibility issues prior to Intel releases, I assumed this situation would be minimal. If we get in the situation where Intel needs to introduce a breaking change and we don't have an opportunity to get ahead of it pre-release then this blurb would be incorrect and break new manual setups for users. This situation wouldn't happen if we point users to a known working version. 

If the extra prerequisites file suggested in my earlier comment is implemented I could replace this change with a reference to that instead.

Reply:
Intel is not breaking compatibility with their releases. So I think saying latest is fine. Do be extra cautious, pointing to the markdown is also a fine.